common/src/main/java/com/wynntils/services/map/Label.java

```diff
@@ -6,19 +6,33 @@
 
 import com.wynntils.utils.StringUtils;
 import com.wynntils.utils.mc.type.Location;
+import com.wynntils.utils.type.RangedValue;
 
 public class Label {
     private final String name;
     private final int x;
     private final int z;
     private final int layer;
-    private final String level;
 
-    public Label(String name, int x, int z, int layer, String level) {
+    // level is provided as a string for some data sources
+    private final String levelString;
+    private final transient RangedValue level;
```

Comment:
How does a ranged value apply? If a place is "level 70", do you want to store that as a range 70-130? Or does the range signify an uncertainty in the lowest level, like "level 30-40", so a more exact value like "level 30" would be the range 30-30?

Otherwise, I think the level should be just an integer, telling the lowest recommended level. That is the usual case for this.

Reply:
(We should actually be able to use this, and e.g. hide everything that is above your level, or mark it as dangerous with a ⚠️ symbol or whatever. That is really a missed opportunity.)

Reply:
I think there is a value in specifying a max level, where the area is no longer interesting to the player.

Reply:
I see what you mean, but I think that is problematic. 

1) I think such a judgement will be much more subjective than specifying a min level. If you go to places above your level, you'll die almost instantly by mobs. Otoh, I can sometimes go to too simple places just because it is more peaceful there. How do you determine that level?

2) The official sources have a min level specified, but not a max level. That means we can not get such data for official sources.

Reply:
Okay. Let's do integers as levels for now.